### PR TITLE
Document Board LEDs

### PR DESCRIPTION
**Pull request by WillB97**
This just documents what the LEDs do to aid in debugging, particularly at helpdesk.

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -63,3 +63,6 @@ VESA
 WEEE
 yocto
 zoloto
+overcurrent
+undervoltage
+SMPS
```
Comment (PeterJCLaw) [inline .spelling:68]: drive-by: Please can we keep these alphabetically sorted within their group

**Review by trickeydan — APPROVED** [review]
